canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -771,19 +771,21 @@ export default class PropertiesController {
 	*
 	* @param {object} propertyId. required.
 	* @param {boolean} showErrors. optional. Set to false to run conditions without displaying errors in the UI
+	* @param {boolean} onlyValidateInvalidFieldDefinition. optional. Set to true to validate "colDoesExists" condition
 	*    Defaults to true to always display errors
 	*/
-	validateInput(propertyId, showErrors = true) {
-		conditionsUtil.validateInput(propertyId, this, showErrors);
+	validateInput(propertyId, showErrors = true, onlyValidateInvalidFieldDefinition = false) {
```

Comment:
Instead of adding a new parameter can we assume of showErrors=false we'd only run the fieldValidation condition?  If we keep the additional parameter we should make it an object instead so we could add more validation if necessary in the future.

Reply:
I can validate invalid field definitions when showErrors= false. But then, I have to define a new variable for displaying warning msgs -

```
let showInvalidFieldDefinitionErrors = false; // new variable for displaying warnings even when showErrors=false
if (!showErrors) {
	const invalidFieldDefinition = validations.find((val) => get(val, "definition.validation.id", "").includes("validField_"));
	validations = (typeof invalidFieldDefinition === "undefined") ? [] : [invalidFieldDefinition];
	showInvalidFieldDefinitionErrors = true;
}


// Updated this condition to show warnings
if (!showErrors && !showInvalidFieldDefinitionErrors) {
	errorMessage.displayError = false;
}
```

Reply:
This approach may not work because I am replacing all validations with "colDoesExists" validations when `showErrors=false`. So it ignores other validations which regressed this - 
Save button should be disabled because required field is empty. But this condition was never checked. 
![Screen Shot 2022-08-18 at 12 11 51 PM](https://user-images.githubusercontent.com/25124000/185475566-ce7c195c-192e-4184-a731-ceb430d97bd3.png)